### ISSUE TITLE
Required resources doesn’t update 

### ISSUE THREAD

**Issue by riveruniversity** [quoid/userscripts#523]
Hello,

I am using the extension on iOS Safari iPad and iPhone. 
I want to clear the cache as instructed but the downloaded JavaScript file does not update. 
> if you want to update the require resources, and the url does not change, you must remove the resources, save, then re-input it

It works when I change the local file name of the user script.

**Comment by riveruniversity** [comment]
I’m sorry for lack of clarification. I am actually removing the `@require` metadata line, saving the script, and re-adding it and saving again as described in your instructions.

**Comment by quoid** [comment]
> @quoid Maybe we should update this note to remove possible ambiguity.

Yeah, I think so. Also maybe a note about iOS vs macOS considering this conversation.

**Comment by riveruniversity** [comment]
@ACTCD I did open the **`popup`** each time after saving the file as you instructed but it didn't work. I will just keep changing the filename of the user script as it's the only way it works for me as for now.

Thank you much.
